### ISSUE TITLE
Ability to set heights and floor offsets for shading groups

### ISSUE THREAD

**Issue by macumber** [NatLabRockies/floorspace.js#289]
Main use case is to specify shading group on ground level then specify entire building height rather than create shading on multiple stories

**Comment by macumber** [comment]
Will we want shading to have roofs?  Seems like that might be useful but probably can't share a roof between spaces and shading.

**Comment by shorowit** [comment]
It would be nice to be able to assign a pitched roof to shadings. 

The pitched roof object is a definition, so if you assign it to, say, two non-adjacent spaces, each space would get a separate roof as defined by the pitched roof object. If the spaces are adjacent, then there is a single roof that spans both spaces. The same logic should work for shadings.

Though this makes me wonder: how do you translate a shading drawn adjacent to a space?

**Comment by macumber** [comment]
Yeah I just did this (added a test) currently there is no intersection/matching of surfaces with shading objects
